alf/examples/actor_critic.py

```diff
@@ -0,0 +1,209 @@
+        # Params for train
+        train_interval=20,
+        num_steps_per_iter=100,
+        num_iterations=1000,
+        learning_rate=5e-5,
+        # Params for checkpoints
+        checkpoint_interval=10000,
+        # Params for summaries and logging
+        log_interval=50,
+        summary_interval=50,
+        summaries_flush_secs=1,
+        use_tf_functions=True,
+        debug_summaries=False,
+        summarize_grads_and_vars=False):
+    """A simple train and eval for ActorCriticPolicy."""
+
+    if root_dir is None:
+        raise AttributeError('train_eval requires a root_dir.')
+
+    root_dir = os.path.expanduser(root_dir)
+    train_dir = os.path.join(root_dir, 'train')
+
+    train_summary_writer = tf.summary.create_file_writer(
+        train_dir, flush_millis=summaries_flush_secs * 1000)
+    train_summary_writer.set_as_default()
+
+    global_step = tf.Variable(
+        0, dtype=tf.int64, trainable=False, name="global_step")
+
+    with tf.summary.record_if(
+            lambda: tf.math.equal(global_step % summary_interval, 0)):
+        tf.random.set_seed(random_seed)
+        py_env = parallel_py_environment.ParallelPyEnvironment(
+            [lambda: env_load_fn(env_name)] * num_parallel_environments)
+        tf_env = tf_py_environment.TFPyEnvironment(py_env)
+        optimizer = tf.optimizers.Adam(learning_rate=learning_rate)
+
+        if use_rnns:
+            actor_net = ActorDistributionRnnNetwork(
+                tf_env.observation_spec(),
+                tf_env.action_spec(),
+                input_fc_layer_params=actor_fc_layers,
+                output_fc_layer_params=None)
+            value_net = ValueRnnNetwork(
+                tf_env.observation_spec(),
+                input_fc_layer_params=value_fc_layers,
+                output_fc_layer_params=None)
+        else:
+            actor_net = ActorDistributionNetwork(
+                tf_env.observation_spec(),
+                tf_env.action_spec(),
+                fc_layer_params=actor_fc_layers)
+            value_net = ValueNetwork(
+                tf_env.observation_spec(), fc_layer_params=value_fc_layers)
+
+        policy = ActorCriticPolicy(
+            actor_network=actor_net,
+            value_network=value_net,
+            optimizer=optimizer,
+            time_step_spec=tf_env.time_step_spec(),
+            action_spec=tf_env.action_spec(),
+            train_interval=train_interval,
+            debug_summaries=debug_summaries,
+            train_step_counter=global_step)
+
+        checkpointer = tfa_common.Checkpointer(
+            ckpt_dir=os.path.join(train_dir, 'policy'),
+            policy=policy,
+            global_step=global_step)
+
+        train_metrics = [
+            tf_metrics.NumberOfEpisodes(),
+            tf_metrics.EnvironmentSteps(),
+            tf_metrics.AverageReturnMetric(),
+            tf_metrics.AverageEpisodeLengthMetric(),
+        ]
+        driver = PyDriver(
```

Comment:
alternative way to implement a new driver: e.g. on policy driver, async driver.

Reply:
> alternative way to implement a new driver: e.g. on policy driver, async driver.

Yes, that might be a good idea, which involves merging driver and TrainingPolicy. I can do it in another PR.